homeassistant/components/samsungtv/bridge.py

```diff
@@ -328,12 +346,15 @@ def device_info(self) -> dict[str, Any] | None:
 
         return None
 
-    def _send_key(self, key: str) -> None:
+    def _send_key(self, key: str, key_type: str | None = None) -> None:
         """Send the key using websocket protocol."""
         if key == "KEY_POWEROFF":
             key = "KEY_POWER"
         if remote := self._get_remote():
-            remote.send_key(key)
+            if key_type == "run_app":
```

Comment:
Why do we overload the `_send_key` method to run apps ? The media player knows it's running an app and could maybe call `run_app` directly ?

Reply:
That is because the primary `send_key` method in the bridge has a retry mechanism that I didn't want to bypass.
I planned to rename the method to make it more obvious (`send_key` to `send_command`) in a separate PR.